### ISSUE TITLE
New release to fix build of reagents?

### ISSUE THREAD

**Issue by edwintorok** [ocaml-multicore/kcas#10]
Now that multicore is closer to becoming reality (https://github.com/ocaml/ocaml/pull/10831) it'd be good to be able to easily install/use the rest of the libraries around it.
https://github.com/ocaml-multicore/reagents/issues/25 says a new kcas is needed to build, and I confirm that is still the case with latest OCaml 4.12.0+domains+effects and the multicore opam repo.

```
opam remote -v
[NOTE] These are the repositories in use by the current switch. Use '--all' to see all configured repositories.

<><> Repository configuration for switch 4.12.0+domains+effects <><><><><><><><>
 1 multicore git+https://github.com/ocaml-multicore/multicore-opam.git
 2 default   https://opam.ocaml.org
```

By default you get a compile error like this (if you install the missing `jbuilder` dependency):
```
#=== ERROR while compiling reagents.0.3.0 =====================================#
# context     2.1.1 | linux/x86_64 | ocaml-variants.4.12.0+domains+effects | git+https://github.com/ocaml-multicore/multicore-opam.git
# path        /var/tmp/edwin-caches/.opam/4.12.0+domains+effects/.opam-switch/build/reagents.0.3.0
# command     /var/tmp/edwin-caches/.opam/opam-init/hooks/sandbox.sh build jbuilder build -p reagents -j 8
# exit-code   1
# env-file    /var/tmp/edwin-caches/.opam/log/reagents-112796-f96fc6.env
# output-file /var/tmp/edwin-caches/.opam/log/reagents-112796-f96fc6.out
### output ###
# [...]
#       ocamlc reagents/.reagents.objs/reagents__Offer.{cmo,cmt} (exit 2)
# (cd _build/default && /var/tmp/edwin-caches/.opam/4.12.0+domains+effects/bin/ocamlc.opt -w -40 -g -bin-annot -I reagents/.reagents.objs -I /var/tmp/edwin-caches/.opam/4.12.0+domains+effects/lib/kcas -I /var/tmp/edwin-caches/.opam/4.12.0+domains+effects/lib/lockfree -no-alias-deps -open Reagents__ -o reagents/.reagents.objs/reagents__Offer.cmo -c -impl reagents/offer.ml)
# File "reagents/offer.ml", line 42, characters 17-28:
# 42 |   let get_id r = Kcas.get_id r
#                       ^^^^^^^^^^^
# Error: Unbound value Kcas.get_id
#     ocamlopt reagents/.reagents.objs/reagents__Offer.{cmx,o} (exit 2)
# (cd _build/default && /var/tmp/edwin-caches/.opam/4.12.0+domains+effects/bin/ocamlopt.opt -w -40 -g -I reagents/.reagents.objs -I /var/tmp/edwin-caches/.opam/4.12.0+domains+effects/lib/kcas -I /var/tmp/edwin-caches/.opam/4.12.0+domains+effects/lib/lockfree -no-alias-deps -open Reagents__ -o reagents/.reagents.objs/reagents__Offer.cmx -c -impl reagents/offer.ml)
# File "reagents/offer.ml", line 42, characters 17-28:
# 42 |   let get_id r = Kcas.get_id r
#                       ^^^^^^^^^^^
# Error: Unbound value Kcas.get_id
```

The workaround mentioned in the above issue still works:
```
opam pin add kcas --dev-repo
```

Could you make a new release to allow these to be able to installed/built together?
```

**Comment by kayceesrk** [comment]
Thanks for the bug report. `kcas` and `reagents` are not actively maintained now as the multicore team does not have the resources to do so. We will have to come back to this after the 5.00 release.

**Comment by bartoszmodelski** [comment]
I believe this is now resolved with 0.1.5 release in opam.
